### DISCUSSION TITLE
Startup on english/romaji mode?

### DISCUSSION THREAD

**Discussion by beansmayawait** [google/mozc#700] [Q&A]
Is there a way to startup on english/romaji mode? I find myself typing gibberish hiragana into terminal after reboots. It would be nice to not have to remember to press ZEN/HAN every bootup.

I start fcitx from i3wm's config file. in fcitx-configtool mozc is the only input method.

**Comment by hiroyuki-komatsu** [comment]
Hi beansmayawait,

For IBus, you can use `active_on_launch` or `composition_mode` to change the initial input mode.

https://github.com/google/mozc/blob/master/docs/configurations.md#ibus

For Fcitx, I think there are ways to do that too, although this project/repository does not directly cover Fcitx.
Thanks,

**Comment by HubKing** [comment]
I think the easiest way is adding two methods: English (default) and Mozc. 

PS: Also, you can select the initial mode in Mozc's settings. You can use "Direct".
![image](https://user-images.githubusercontent.com/10900489/229302741-a783b802-a6d7-4113-9601-dcec0e24abdf.png)


Also, if you use IBus, it starts with the English mode (_A) by default. I hate that; I want Hiragana by default, and that is why I switched to Fcitx, even though I use Gnome, whose default IME is IBus.

**Reply by shinya17** [reply]
I use IBus and i updated the config file as described in the above comment by @hiroyuki-komatsu and it solved my problem. Did it not work for you ?
